src/main/java/org/jabref/gui/search/GlobalSearchResultDialog.java

```diff
@@ -76,5 +77,14 @@ private void initialize() {
                     ? IconTheme.JabRefIcons.KEEP_ON_TOP.getGraphicNode()
                     : IconTheme.JabRefIcons.KEEP_ON_TOP_OFF.getGraphicNode());
         });
+
+        getDialogPane().getScene().getWindow().addEventHandler(WindowEvent.WINDOW_SHOWN, event -> {
+            getDialogPane().setPrefHeight(viewModel.searchWindowHeight());
+            getDialogPane().setPrefWidth(viewModel.searchWindowWidth());
+        });
```

Comment:
`bindBidirectional` to SearchPreferences?

Reply:
I am not sure what you mean. Could you please elaborate?